daprdocs/content/en/operations/resiliency/policies.md

```diff
@@ -132,4 +130,147 @@ spec:
     apps:
       appB: # app-id of the target service
         retry: retryForever
-```
\ No newline at end of file
+```
+
+## Setting Default Policies
+
+In resiliency you can set default policies, which can have a broader scope. This is done through reserved keywords that let Dapr know when to apply the given policy. There are 3 default policies types: 
+
+- DefaultRetryPolicy
+- DefaultTimeoutPolicy
+- DefaultCircuitBreakerPolicy
+
+If these policies are defined, they would be used for every operation to a service, application, or component. They can also be modified to be more specific through the usage of additional keywords. The specific policies follow the following pattern, `Default%sRetryPolicy`, `Default%sTimeoutPolicy`, and `Default%sCircuitBreakerPolicy`. Where the `%s` is replaced by the new target of the policy. Below you can see a table of all possible default policy keywords and how they translate into a policy name.
+
+| Keyword                        | Target Operation                                     | Example Policy Name                                       |
```

Comment:
```suggestion
| Keyword                        | Target Operation                                     | Policy Name                                       |
```

Reply:
I kept this as `Example` because it isn't a complete list of all possible names. Just an example of how that keyword applies into the names. If you still think I should remove example, let me know. Or I can add the complete list of everything, but my concern there was that it would be a large number and people may just glaze over while reading it.